stacks-coordinator/src/bitcoin_node.rs

```diff
@@ -177,13 +184,19 @@ impl LocalhostBitcoinNode {
     }
 
     fn import_address(&self, address: &BitcoinAddress) -> Result<(), Error> {
-        let address = address.to_string();
         debug!("Importing address {}...", address);
+        // Create a descriptor using a Bech32 (segwit) Pay-to-Witness Public Key Hash (P2WPKH) address.
+        let desc = format!("wpkh({})", address);
```

Comment:
I suspect this is incorrect. We're using a taproot address and not a p2wpkh address. I guess we don't have any transactions on-chain that we have verified this with right? I.e. our only verification so far is that the call succeeds?

Reply:
Update: It seems like we should be good if we simply do it like this
```suggestion
        // Create a descriptor using a Bech32 (segwit) Pay-to-Taproot (P2TR) address.
        let desc = format!("tr({})", address);
```

Reply:
I committed a few shell scripts I used to play around with this and understand these RPC calls better. They are under `/misc`.

Reply:
Feel free to commit this change.

Reply:
Oh I thought I already did that :sweat_smile: Done now :heavy_check_mark: